### PR TITLE
ENH: Document --no-color option, respect NO_COLOR environment variable

### PR DESCRIPTION
**Pull request by rwblair**

### CODE REVIEW

**Comment by rwblair** [comment]
Cannot figure out why the `--no-color` flag doesn't work in circleci jest run. I ssh'd into the instance and ran the same command that child_process.spawn calls and it worked as expected. Tried inserting `--no-colors` as well.

File: bids-validator/bin/bids-validator [hunk 1]
```diff
@@ -25,6 +25,9 @@ const argv = require('yargs')
   .describe('verbose', 'Log more extensive information about issues')
   .boolean('json')
   .describe('json', 'Output results as JSON')
+  .boolean('no-color')
+  .describe('no-color', 'Disable colors in output text.')
+  .default('no-color', false)
```
Comment (effigies) [inline bids-validator/bin/bids-validator:30]: Where is the effect of passing `--no-color`?

Comment (rwblair) [reply bids-validator/bin/bids-validator:30]: https://github.com/Marak/colors.js#enablingdisabling-colors

Built in from the colors library.

Comment (effigies) [reply bids-validator/bin/bids-validator:30]: Ah, so this is just documentation?

Comment (rwblair) [reply bids-validator/bin/bids-validator:30]: Yee that was the idea.

Comment (effigies) [reply bids-validator/bin/bids-validator:30]: Okay. I'm going to try to install locally to test, but otherwise I'd say this is good to go.

**Pull request merged**